packages/sw-broadcast-cache-update/src/lib/behavior.js

```diff
@@ -64,27 +65,22 @@ import {defaultHeadersToCheck, defaultSource} from './constants';
  *     cacheName
  *   });
  * }
+ *
+ * @memberof module:sw-broadcast-cache-update
  */
 class Behavior {
   /**
-   * Creates a new `Behavior` instance, which is used to compare two
-   * [Responses](https://developer.mozilla.org/en-US/docs/Web/API/Response)
-   * and use the {@link https://developers.google.com/web/updates/2016/09/broadcastchannel|Broadcast Channel API}
-   * to notify interested parties when those Responses differ.
-   *
-   * For efficiency's sake, the underlying response bodies are not compared;
-   * only specific response headers are checked.
+   * To instantiate this Class you must provide a channel name which
+   * will be used to when creating a broadcast channel to dispatch messages on.
    *
-   * @param {Object} input The input object to this function.
+   * @param {Object} input
    * @param {string} input.channelName The name that will be used when creating
-   *        the [`BroadcastChannel`](https://developer.mozilla.org/en-US/docs/Web/API/BroadcastChannel/BroadcastChannel).
+   *        the `BroadcastChannel`.
    * @param {Array<string>} input.headersToCheck A list of headers that will be
-   *        used to determine whether the `Response`s differ. If not provided,
-   *        the values `['content-length', 'etag', 'last-modified']` are used.
-   * @param {string} input.source An attribution value that will be used in the
-   *        broadcast message to indicate where the update originated. If not
-   *        provided, a
-   *        {@link constants#defaultSource|default value} will be used.
+   *        used to determine whether the `Response`s differ. Defaults to
```

Comment:
You've been inconsistent about formatting "Response". If it's a specific object use backticks. If you're generically referring to "a response", remove the backticks and lower case the R. Please check throughout.